platform/engineering/design-docs/2022-03-11-schedule-pending-appt.md

```diff
@@ -0,0 +1,156 @@
+                  "request": "false",
+                  "request_reason": "Does not provide chosen service",
+                  "direct": "false",
+                  "direct_reason": "nil"   
+               }
+            },
+            {
+               "facility_info": "test"
+            }
+         ],
+         "cc_providers": [
+            {
+               "name": "Ashinoff, Stephen",
+               "address": "31-75 23rd st, Long Island City, NY 11106",
+               "distance": "0.7 Miles"
+            }
+         ],
+         "va":[
+            {
+               "name": "Cheyenne VA Medical Center",
+               "city": "Cheyenne",
+               "state": "WY",
+               "cerner": "true",
+               "eligible": {
+                  "request": "false",
+                  "request_reason": "Non-primary facility with no visit within 12-24 months",
+                  "direct": "false", 
+                  "direct_reason": "nil"
+               }
+            },
+            {
+               "name": "Dayton VA Medical Center",
+               "city": "Dayton",
+               "state": "OH",
+               "cerner": "true",
+               "eligible": {
+                  "request": "true",
+                  "request_reason": "nil",
+                  "direct": "false", 
+                  "direct_reason": "nil"
+               }
+            }
+         ]
+      }
+   ]
+}
+```
+Note: Direct scheduling will be hardcoded to false until direct schedule feature is added.
+
+### Code Location
+In `vets-api`, a controller will be created called `schedule_appointment_request_controller`. Serializers, models, and services will be placed in their corresponding folders in `modules/mobile/app`
+
+### Testing Plan
+Specs will be written for each service, adapter and controller, to provide full code coverage.
+
+### Logging
+Logging will be done similar to how other Mobile API endpoints are logged, through the Rails Logger and Sentry.
+
+### Debugging
+Debugging will be done similar to how other Mobile API endpoints are debugged, utilizing logs and Sentry to diagnose issues and testing individual endpoints to isolate issues.
+
+### Caveats
+To be determined.
+
+### Security Concerns
+N/A
+
+### Privacy Concerns
+N/A
+
+### Open Questions and Risks
+Will aggregating all external requests at the beginning of the process cause long load times? If so, we may want to precache results. 
+
+Are endpoints we'll be consuming reliable enough to have a good user experience?
+
+### Work Estimates
+TODO
+
+### Alternatives
+Rather then relying on other endpoints, we could create these features from the ground up to ensure we have full control of all parts of development and maintenance but it has been a long established objective of the Mobile API to reuse backend integrations wherever possible and to not add complexity to an already complex system. Creating redundant logic would not serve either of these objectives
```

Comment:
```suggestion
Rather than relying on other endpoints, we could create these features from the ground up to ensure we have full control of all parts of development and maintenance but it has been a long established aim of the Mobile API to reuse backend integrations wherever possible and to not add complexity to an already complex system. Creating redundant logic would not serve either of these objectives.
```